specification/subscription/resource-manager/Microsoft.Subscription/stable/2021-10-01/examples/getSubscriptionOperation.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "operationId": "e4b8d068-f574-462a-a76f-6fa0afc613c9",
+    "api-version": "2021-10-01"
+  },
+  "responses": {
+    "202": {
```

Comment:
GET operation should not return 202.

Reply:
Hi @ms-henglu , we had a discussion with Zizhou and is fine to suppress this issue, and return this get with 200 and 202 we had this same api in 2019-10-01-preview version.

Reply:
@ms-henglu Just to add some background, from the discussion we had last week, they already have deployed the manifest and this API is running in production.

@abhaygill338 Your lint error is still not suppressed properly, please check the error, once the suppression is there, it'll be reviewed